docs/docs/rest-api/public/api/v2/tasks.raml

```diff
@@ -1,9 +1,10 @@
 get:
   description:
-    List all running tasks.
+    List all tasks.
   is: [ secured ]
   queryParameters:
     status:
+      required: false
```

Comment:
Please add description about the behavior when status parameter is not provided.